rasa/constants.py

```diff
@@ -29,16 +29,16 @@
 DEFAULT_RASA_X_PORT = 5002
 DEFAULT_RASA_PORT = 5005
 
-DOCS_URL_POLICIES = DOCS_BASE_URL + "/core/policies/"
+DOCS_URL_POLICIES = DOCS_BASE_URL + "/policies"
 DOCS_URL_TEST_STORIES = DOCS_BASE_URL + "/testing-your-assistant"
-DOCS_URL_ACTIONS = DOCS_BASE_URL + "/core/actions/"
-DOCS_URL_CONNECTORS = DOCS_BASE_URL + "/user-guide/connectors/"
-DOCS_URL_EVENT_BROKERS = DOCS_BASE_URL + "/api/event-brokers/"
+DOCS_URL_ACTIONS = DOCS_BASE_URL + "/actions"
+DOCS_URL_CONNECTORS = DOCS_BASE_URL + "/connectors/"
+DOCS_URL_EVENT_BROKERS = DOCS_BASE_URL + "/event-brokers"
 DOCS_URL_PIKA_EVENT_BROKER = DOCS_URL_EVENT_BROKERS + "#pika-event-broker"
-DOCS_URL_TRACKER_STORES = DOCS_BASE_URL + "/api/tracker-stores/"
-DOCS_URL_PIPELINE = DOCS_BASE_URL + "/nlu/choosing-a-pipeline/"
-DOCS_URL_COMPONENTS = DOCS_BASE_URL + "/nlu/components/"
-DOCS_URL_MIGRATION_GUIDE = DOCS_BASE_URL + "/migration-guide/"
+DOCS_URL_TRACKER_STORES = DOCS_BASE_URL + "/tracker-stores"
+DOCS_URL_PIPELINE = DOCS_BASE_URL + "/tuning-your-model"
+DOCS_URL_COMPONENTS = DOCS_BASE_URL + "/components"
+DOCS_URL_MIGRATION_GUIDE = DOCS_BASE_URL + "/migrate-from/"
```

Comment:
This is different -- migraation guide is migrating from one version to another, migrate from is migrating from other tools to rasa